AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/OutgoingMutationQueue.swift

```diff
@@ -138,6 +138,7 @@ final class OutgoingMutationQueue: OutgoingMutationQueueBehavior {
                        mutationEventPublisher: MutationEventPublisher) {
         log.verbose(#function)
         self.api = api
+        dispatchOutboxStatusEvent(isEmpty: operationQueue.operationCount == 0)
```

Comment:
`operationQueue.operationCount` is always 0 right here i think, because we haven't `request(.max(1))` yet so no mutation events recieved, and no mutation event added to the operation queue